dbus/methods_test.go

```diff
@@ -132,6 +132,19 @@ func runStopUnit(t *testing.T, conn *Conn, trTarget TrUnitProp) error {
 	return nil
 }
 
+func getJobStatusIfExists(jobs []JobStatus, jobName string) *JobStatus {
+	for _, j := range jobs {
+		if j.Unit == jobName {
+			return &j
+		}
+	}
+	return nil
+}
+
+func isJobStatusEmpty(job JobStatus) bool {
+	return job.Id == 0 && job.Unit == "" && job.JobType == "" && job.Status == "" && job.JobPath == "" && job.UnitPath == ""
```

Comment:
Nit: as unlikely as it could be, an `Id = 0` I think is a valid one which may result in test flakes, right? The rest of the condition seems fine on its own.

Reply:
Just to clarify, we should remove check for Id, right? 
Also, should it be `&&` or `||` between rules?

Reply:
Ah, I misread this check and already assumed the conditions where ORed. I think I agree with your current way of checking whether **all** fields are defaults. Just disregard my previous comment, sorry for the noise :)